examples/aws-cloudwatch/dc-aws-cloudwatch.yml

```diff
@@ -32,7 +32,7 @@ mysql-createdb:
     - juttle-engine
   volumes:
     - ${PWD}/aws-cloudwatch/create_aws_db.sql:/opt/incoming/create_aws_db.sql
-  command: bash -c "sleep 10 && mysql -u root -h examples_mysql_1 --password=my-juttle-password < /opt/incoming/create_aws_db.sql && tail -f /dev/null"
+  command: bash -c "sleep 20 && mysql -u root -h examples_mysql_1 --password=my-juttle-password < /opt/incoming/create_aws_db.sql && tail -f /dev/null"
```

Comment:
yeah the sleeps are necessary because a container can start before it is really ready to accept connections, etc. Here's a discussion on that: https://github.com/docker/compose/issues/374.

In docker-compose 1.6 there is an events framework where containers can explicitly signal their readiness to each other in some way, and that should allow me to get rid of the sleeps. However, docker-compose 1.6 is really new and I haven't tried it out yet to see if it works.